.travis.yml

```diff
@@ -43,6 +43,36 @@ jobs:
         secure: "HkgYgawkr/hkg2vURHGOb/JmF6U1e71QQsd0HXJ1UJh6WBiFJOrjomCMHZGHnpyopdRna++up8dISBqM2X+EiLAr9yWdRXy72oMP9X42M0ccnja/3E5KYYXMkETS50JnUhMCOt2xRZW5/ojqrHukqRo0YIjIu+P0U0VsZ4QIsng="
 
     # Docker Build #
+    - stage: Build
+      env:
+         - NODE_VERSION: "10"
+         - VARIANT: "default"
+
+    - stage: Build
+      env:
+         - NODE_VERSION: "10"
+         - VARIANT: "alpine"
+
+    - stage: Build
+      env:
+         - NODE_VERSION: "10"
+         - VARIANT: "onbuild"
+
+    - stage: Build
+      env:
+         - NODE_VERSION: "10"
+         - VARIANT: "slim"
+
+    - stage: Build
+      env:
+         - NODE_VERSION: "10"
+         - VARIANT: "stretch"
+
+    - stage: Build
+      env:
+         - NODE_VERSION: "10"
+         - VARIANT: "wheezy"
```

Comment:
Is it worth keeping a wheezy build around? It goes EOL in May:
https://www.debian.org/releases/wheezy/

Reply:
No need to create a new one at least. We have #567 open for removing the build of the old ones as well

Reply:
Looks like this got missed -- `10/wheezy` should be axed before we merge https://github.com/docker-library/official-images/pull/4289, right?